themes/theme-ios11/pages/index.jsx

```diff
@@ -38,49 +39,53 @@ import locale from '../locale';
 import themeApi from '../themeApi';
 import * as routes from './routes';
 
+const themeConfigParse = new ConfigParser(themeConfig).parse();
```

Comment:
Same typo as above. `themeConfigParse` should be `themeConfigParsed`

Reply:
Fixed